symengine/cwrapper.cpp

```diff
@@ -160,7 +160,7 @@ TypeID basic_get_type(const basic s)
     return static_cast<TypeID>(s->m->get_type_code());
 }
 
-void symbol_set(basic s, char *c)
+void symbol_set(basic s, const char *c)
```

Comment:
@isuruf exactly. That's why I think that `const char *` is enough. It's more readable and it does what we need it to do.

@abinashmeher999 the const is useful to check *your* code (not the user's code) that it doesn't accidentally modify it.